### DISCUSSION TITLE
why so much weight in the containers 'Operating System'?

### DISCUSSION THREAD

**Discussion by enreda-net** [tteck/Proxmox#3539] [Ideas]
It's a shame that the LXC OSes available for installation on tteck take up so much space.
Debian tteck is 30% heavier than the Debian container available on proxmox.
Ubuntu tteck is 20% heavier than the Ubuntu container available on proxmox.
Comparison after upgrading, of course.

I haven't looked at the difference in installed packages but I have seen higher RAM usage in the tteck containers.

There must be a reason for this, but my decision has been not to use the tteck containers.

**Comment by tteck** [comment]
The shame is you not looking for yourself before posting.
It starts with the standard templates, and installs
https://github.com/tteck/Proxmox/blob/631ef46466dfc8feffe1bd358ee5289850f89b1d/install/debian-install.sh#L17-L19
https://github.com/tteck/Proxmox/blob/631ef46466dfc8feffe1bd358ee5289850f89b1d/install/ubuntu-install.sh#L17-L19

**Comment by enreda-net** [comment]
The shame is that you waste more time answering than trying to understand what you read.
________________________________
De: tteckster ***@***.***>
Enviado: domingo, 18 de agosto de 2024 5:19
Para: tteck/Proxmox ***@***.***>
Cc: oantonioo ***@***.***>; Author ***@***.***>
Asunto: Re: [tteck/Proxmox] why so much weight in the containers 'Operating System'? (Discussion #3539)


The shame is you not looking for yourself before posting.
It starts with the standard templates, and installs
https://github.com/tteck/Proxmox/blob/631ef46466dfc8feffe1bd358ee5289850f89b1d/install/debian-install.sh#L17-L19
https://github.com/tteck/Proxmox/blob/631ef46466dfc8feffe1bd358ee5289850f89b1d/install/ubuntu-install.sh#L17-L19
